src/main/java/com/tmartsyniak/learning/shapes/task1/Main.java

```diff
@@ -0,0 +1,28 @@
+package com.tmartsyniak.learning.shapes.task1;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException, ClassNotFoundException {
+
+        List<Figure> sourceFigures = new ArrayList<>();
+        FiguresIOUtil util = new FiguresIOUtil();
+        String file = "src/main/java/com/tmartsyniak/learning/shapes/task1/test.ser";
```

Comment:
Variables should be declared as close as possible to their usage. You declared util and file variables and used them 4 lines below.